src/main/java/org/folio/circulation/resources/RequestCollectionResource.java

```diff
@@ -194,4 +188,14 @@ private ProxyRelationshipValidator createProxyRelationshipValidator(
       singleValidationError("proxyUserId is not valid",
         PROXY_USER_ID, representation.getString(PROXY_USER_ID)));
   }
+
+  private String getRequestId(RoutingContext routingContext) {
+    return routingContext.request().getParam("id");
+  }
+
+  private UniqRequestValidator getUniqRequestValidator() {
+    return new UniqRequestValidator(
+      request -> singleValidationError("This requester already has an open request for this item")
```

Comment:
Validation failures should include one or more parameter that they relate to.

In this case, that is probably the `itemId` and the `userId` of the request.

Reply:
Changed